package.json

```diff
@@ -24,7 +24,7 @@
     "index.js"
   ],
   "peerDependencies": {
-    "eslint": "^0.15.0"
+    "eslint": "^0.16.0"
```

Comment:
Seeing how the tests pass between both version, we should be able to use a range and support both libraries.

@MoOx Thoughts?